Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -96,114 +96,31 @@ final class CacheAdvanceTests: XCTestCase {
         XCTAssertEqual(messages, [])
     }
 
-    func test_messages_whenOffsetInFileAtEndOfNewestMessageIsBeyondEndOfNewestMessageButBeforeEndOfFile_throwsFileCorrupted() throws {
```

Comment:
why is this test (and the deleted one below) no longer valid? I think testing these cases is still valuable, but I might be missing something.

Reply:
These tests should be kept. I reverted the last commit, this was deleted then.